crates/autopilot/src/solvable_orders.rs

```diff
@@ -148,6 +151,18 @@ impl SolvableOrdersCache {
         let removed = counter.checkpoint("banned_user", &orders);
         invalid_order_uids.extend(removed);
 
+        let orders = if self
```

Comment:
I think we should only deprecate the order creation at the cut-off point but we still might have some open market orders in the system which should still be settled.

Reply:
> this PR is intended to land together with CIP38 code (which makes assumptions that there are no market orders

Should then both of these behaviors be controlled by the same flag to avoid possible misconfiguration?

Reply:
I removed this handling (let's hope we wont need it). 
Since forwarding `valid_to` and `min_validity` is not trivial and results in too many args moved around, I will just set the CIP38 activation to X and rejection of market orders to X-6 hours (this should be enough for all market orders to either be filled or to expire). Since I will be responsible for creating the infra PR for these changes, I think there is no any risk here.